packages/@aws-cdk/aws-eks/README.md

```diff
@@ -811,6 +812,84 @@ const chart2 = cluster.addHelmChart(...);
 chart2.node.addDependency(chart1);
 ```
 
+#### CDK8s Charts
+
+[CDK8s](https://cdk8s.io/) is an open-source library that enables Kubernetes manifest authoring using familiar programming languages. It is founded on the same technologies as the AWS CDK such as [`constructs`](https://github.com/aws/constructs) and [`jsii`](https://github.com/aws/jsii).
+
+> To learn more about cdk8s, visit the [Getting Started](https://github.com/awslabs/cdk8s/tree/master/docs/getting-started) tutorials.
+
+The EKS module natively integrates with cdk8s and allows you to apply cdk8s charts on AWS EKS clusters via the `cluster.addCdk8sChart` method.
+
+To get started, add the `cdk8s` dependency to your `package.json` file.
+
+```json
+dependencies: {
+  "cdk8s": "^0.29.0"
+}
+```
+
+> Note that the version of `cdk8s` must be `>=0.29.0`.
+
+Next, install the `cdk8s-cli`:
```

Comment:
I am wondering if at the entry level we should just demonstrate how to use cdk8s-plus and skip over importinga d CLI installation.

Then in a deeper readme section we can talk about using imported resources.

Reply:
👍